### PR TITLE
Update paths to main files

### PR DESCRIPTION
**Pull request by rallin**

### CODE REVIEW

**Comment by maxigimenez** [comment]
:+1:

File: bower.json [hunk 1]
```diff
@@ -2,10 +2,8 @@
   "name": "angular-google-places-autocomplete",
   "version": "0.2.4",
   "main": [
-      "/src/autocomplete.js",
-      "/src/autocomplete.css",
-      "/dist/autocomplete.min.js",
-      "/dist/autocomplete.min.css"
```
Comment (maxigimenez) [inline bower.json]: Actually you can leave the dist folder and remove the src, to consume the minified versions.

Comment (rallin) [inline bower.json]: According to this specification (https://github.com/bower/bower.json-spec#main), main section should included full version of files such as sources or compiled sources(from CoffeeScript, etc.)

**Comment by kuhnza** [comment]
Thanks @rallin, I'm not getting much time to maintain this right now so I've made you a contributor.

**Pull request closed without merging**